### PR TITLE
Add travis and string validation

### PR DESCRIPTION
**Pull request by sostopher**
## Description
Develop what, but also why this PR exists. The "Why" is as much, if not more important than the "What". I use this approach very often when I write commit messages, because if the code is clear enough it should already answer the "what" question. The "why" is often more subtle.

## CCs
@zendesk/apps-migration
@zendesk/i18n

## Risks
* [low] Adding some stuff

### CODE REVIEW

**Comment by hiroiguchi** [comment]
Thank you Chris!!

@hanli625

**Comment by sostopher** [comment]
Ready for review. Passing YML validation and tests on Travis

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+language: node_js
+node_js:
+  - "8"
+before_install:
+  - bundle config zdrepo.jfrog.io $ARTIFACTORY_USERNAME:$ARTIFACTORY_API_KEY
+script:
+  - npm run lint && npm run build && npm run test
```
Comment (keitheous) [inline .travis.yml]: As we have discussed this morning, We will probably have to split this line

**Comment by hoangtrvu** [comment]
👍 on 🍏

**Pull request merged**